src/auro-flight-main.js

```diff
@@ -73,6 +74,53 @@ class AuroFlightMain extends LitElement {
     return localizedTime;
   }
 
+  /**
+   * @private
+   * @param {string} station Airport code ex: SEA.
+   * @returns Mutated string.
+   */
+  readStation(station) {
+    return Array.from(station).join(' ');
```

Comment:
@aesthetiques Same question as the comment left in the auro-flight-header.js. I don't think we need to abstract this logic into its own function but that is IMO

Reply:
Same reply as above, but I see the point. Being too procedural all the time is a detriment to reading code. But repeating the same function between components is a little annoying. 

You 'could' create a utils file and put the function in there and then reference that in each component.

Reply:
I like this one for the repeated uses, Stephanie and I talked it over yesterday.